runtime/common/src/precompile/dex.rs

```diff
@@ -145,24 +141,20 @@ where
 					path, supply_amount
 				);
 
-				let value = <module_dex::Pallet<Runtime> as DEXManager<Runtime::AccountId, Balance, CurrencyId>>::get_swap_amount(&path, SwapLimit::ExactSupply(supply_amount, Balance::MIN))
+				// If get_swap_amount fail, return 0.
```

Comment:
need to make sure we have some docs on predeploy to require caller to handle the case when the return is zero